common/persistence/src/main/resources/trl-value-mapping-1.15.yaml

```diff
@@ -0,0 +1,25 @@
+transmission-risk-value-mapping:
+  -
+    transmission-risk-level: 1
+    transmission-risk-value: 0
+  -
+    transmission-risk-level: 2
+    transmission-risk-value: 0
```

Comment:
those will be then the ones, that will be filtered out during `submission`, right?

Reply:
@hilmarf  yes